basalt/sdk/datasetsdk.py

```
@@ -2,6 +2,7 @@
 SDK for interacting with Basalt datasets
 """
 from typing import Dict, List, Optional, Tuple, Any
+import asyncio
```

Flake8 (7.2.0)

[error] 'asyncio' imported but unused

(F401)